elbepack/initvmaction.py

```diff
@@ -674,10 +836,14 @@ class SubmitAction(InitVMAction):
     def __init__(self, node):
         InitVMAction.__init__(self, node)
 
-    def execute(self, _initvmdir, opt, args):
-        try:
-            system(f'{sys.executable} {elbe_exe} initvm ensure')
-        except subprocess.CalledProcessError:
+    def execute(self, initvmdir, opt, args):
+        # In case of QEMU mode, we need to forward the additional parameters.
+        additional_params = []
+        if opt.qemu_mode:
+            additional_params = ['--qemu', f'--directory={initvmdir}']
+
+        ps = run_elbe(['initvm', 'ensure'].extend(additional_params), capture_output=True, encoding='utf-8')
```

Comment:
`list.extend()` does not return the extended list.

Either use `[] + additional_params` or `[*additional_params]`.

Do we need to capture the output?